src/olympia/conf/prod/settings.py

```diff
@@ -85,11 +85,11 @@
 
 
 # Celery
-BROKER_URL = env('BROKER_URL')
+CELERY_BROKER_URL = env('CELERY_BROKER_URL')
 
-CELERY_IGNORE_RESULT = True
-CELERY_DISABLE_RATE_LIMITS = True
-BROKER_CONNECTION_TIMEOUT = 0.5
+CELERY_TASK_IGNORE_RESULT = True
+CELERY_WORKER_DISABLE_RATE_LIMITS = True
+CELERY_BROKER_CONNECTION_TIMEOUT = 0.5
```

Comment:
nit: I think some of the above settings are redundant because they're the same as `settings_base`.  Eventually it would be nice if the `/conf/` files _only_ contained settings that differ from `settings_base`

Reply:
indeed, there's an issue somewhere but I can't find it right now. I'd rather fix this in a separate pull request and unify things across all boards.